extensions/wikia/AdEngine/js/recovery/helper.js

```diff
@@ -41,11 +44,11 @@ define('ext.wikia.adEngine.recovery.helper', [
 	}
 
 	function isRecoveryEnabled() {
-		return !!(context.opts.sourcePoint && SourcePointTag);
+		return !!(context.opts.sourcePointRecovery && SourcePointTag);
```

Comment:
We'll have to keep old `context.opts.sourcePoint` and remove it after varnish cache has new, better version ;)